### PR TITLE
fix a bunch of issues

### PR DESCRIPTION
**Pull request by MrSerge01**
# Changes
## Added
- Hot-reloading (`bun dev`)

## Changed
- The way the bot executes commands (it takes commands from an array)

## Fixed
- Running /settings (any subcommand) without options crashes the bot
- 4 roles in /user lists 4 roles, but also says "and **1** more"
- Cooldown setting not clarifying what the user has to provide

### CODE REVIEW

File: src/commands/Leaderboard.ts [hunk 1]
```diff
@@ -80,32 +80,32 @@ export default class Leaderboard {
...
-        });
+      await i.update({
+        embeds: [await generateEmbed()],
+        components: [row]
       });
+    });
 
-      collector.on("end", async () => await interaction.editReply({ components: [] }));
-    }
+    console.log("test");
```
Comment (Froxcey) [inline src/commands/Leaderboard.ts]: remove this

**Review by Froxcey — CHANGES_REQUESTED** [review]
Just a minor detail

vibe

File: src/commands/Leaderboard.ts (hunk 1 above)
Comment (MrSerge01) [reply src/commands/Leaderboard.ts]: removed in next commit

File: README.md
Comment (Golem642) [inline README.md:1]: why did you change it back ?

File: src/commands/User.ts [hunk 2]
```diff
@@ -71,7 +71,7 @@ export default class User {
         `**${guildRoles.filter(role => target.roles.cache.has(role.id)).size! - 1}** ${
           memberRoles.length == 1 ? "role" : "roles"
         } • ${memberRoles
-          .slice(0, 5)
+          .slice(0, 3)
```
Comment (Golem642) [inline src/commands/User.ts:74]: why only 3 ?

File: src/events/interactionCreate.ts
Comment (Golem642) [inline src/events/interactionCreate.ts:1]: i don't get it, does this actually work ? (sorry i'm a bit tired)

**Review by Golem642 — CHANGES_REQUESTED** [review]

File: README.md
Comment (MrSerge01) [reply README.md:1]: yk the logo stuff

File: src/commands/User.ts (hunk 2 above)
Comment (MrSerge01) [reply src/commands/User.ts:74]: because its one line, so **3** roles • roles

File: src/events/interactionCreate.ts
Comment (MrSerge01) [reply src/events/interactionCreate.ts:1]: yes it does, I tested

**Review by Froxcey — APPROVED** [review]
Don't see anything wrong, LGTM

vibe

**Review by dimkauzh — APPROVED** [review]
damn who cooked
